packages/component-utilities/src/echartsCanvasDownload.js

```diff
@@ -7,14 +7,71 @@ export default (node, option) => {
 
 	const chart = init(node, 'evidence-light', { renderer: 'canvas' });
 
-	option.animation = false;
+	option.config.animation = false;
+
+	chart.setOption(option.config);
+
+	// Series Color override
+	const applySeriesColors = () => {
```

Comment:
Why is this being duplicated? Could these functions be pulled out into a shared file?